trnpy/trnsys/simulation.py

```diff
@@ -22,8 +21,9 @@ class Simulation:
     @classmethod
     def new(
         cls,
-        trnsys_lib: Union[str, os.PathLike[Any]],
-        input_file: Union[str, os.PathLike[Any]],
```

Comment:
We can be more strict with the typing here... no need to allow anything besides a `str` or concrete `Path`.